### DISCUSSION TITLE
zh/guide/media_server/push_authentication

### DISCUSSION THREAD

**Discussion by giscus** [ZLMediaKit/docs#3] [Announcements]
# zh/guide/media_server/push_authentication

使用ZLMediaKit的MediaServer进程可以实现推流鉴权： image

https://docs.zlmediakit.com/zh/guide/media_server/push_authentication.html

**Comment by wongdi** [comment]
Can the authentication function be simplified for non-professional users in the future, without relying on an independent authentication server? For example, a password field can be added to the configuration file, and the stream+password method can be used to generate an MD5 value to manually generate an authentication field and implement it in the URL in the form of ?auth= md5. This method is at least simpler for users who do not have a basic setup.

>鉴权功能将来能否针对非专业用户增加一个简单实现，不依赖独立的鉴权服务器，比如在配置文件中增加个密码字段，然后采用 stream+密码 的方式，生成md5值来手动生成鉴权字段并在url中以 ?auth= md5 的方式实现，这种方式至少对没有搭建基础的用户比较简单。

`TRANS_BY_GPT4`
